taqueria-vscode-extension/src/lib/pure.ts

```diff
@@ -120,31 +36,31 @@ export class TaqifiedDir {
 	 * Makes a TaqifiedDir from an existing directory
 	 * @param {PathToDir} dir
 	 * @param {I18N} i18n
-	 * @returns {PromiseLike<E_NOT_TAQIFIED|E_STATE_MISSING, TaqifiedDir>}
+	 * @returns {LikeAPromise<TaqifiedDir, E_NOT_TAQIFIED|E_STATE_MISSING>}
 	 */
-	static create(dir: PathToDir, i18n: I18N): PromiseLike<E_NOT_TAQIFIED | E_STATE_MISSING, TaqifiedDir> {
+	static create(dir: PathToDir, i18n: i18n): LikeAPromise<TaqifiedDir, TaqVsxError> {
 		return makeDir(join(dir, '.taq'), i18n)
 			.then(dotTaqDir =>
 				makeFile(join(dotTaqDir, 'config.json'), i18n)
 					// TODO - validate config!
-					.then(pathToConfig => readJsonFile<Config>(i18n, data => (data as unknown as Config))(pathToConfig))
+					.then(pathToConfig => readJsonFile<Config.t>(i18n, data => (data as unknown as Config.t))(pathToConfig))
 					.catch(previous =>
 						Promise.reject({
 							code: 'E_NOT_TAQIFIED',
 							pathProvided: dir,
-							msg: 'The given directory is not taqified as its missing a .taq/config.json file.',
+							msg: "The given directory is not taqified as it's missing a .taq/config.json file.",
```

Comment:
Note, taquified is spelled: `taq'ified`

I think this message should read:
`The current directory is not a Taqueria project. Please run `taq init` to taq'ify it`

Reply:
This is part of the formatting. May be out of scope for this PR?

Reply:
Also needs improvements to wording

Reply:
The change is from another PR. Should be fixed there or in a new PR.